app/src/app/present-round/present-round.component.html

```diff
@@ -8,4 +8,16 @@
 <p>
 (Všechny změny v depeši se ukládají automaticky. Kliknutím na "Odeslat depeši" dáváte najevo, že depeše je
 kompletní a může začít vyhodnocování. Můžete vše editovat i po odeslání, ale může se stát že už se na to ve vyhodnocování nebude brát zřetel. Vyhodnocování také začne po deadlinu i pokud jste depeši neodeslali. Pokud
-potřebujete více času, napište na Slacku.)</p>
\ No newline at end of file
+potřebujete více času, napište na Slacku.)</p>
+<br />
+<app-decret-form *ngFor="let house of controlledVotingRights | async" [house]="house" [roundId]="roundId" [delegateId]="delegateId"></app-decret-form>
+<br />
+<button mat-raised-button color="accent" *ngIf="!(markedDecreeAsSent | async) else decreeSent" (click)="sendDecree()">Odeslat návrh dekretů</button>
+<ng-template #decreeSent>
+  <p><strong>Návrh dekretů odeslán</strong></p>
+</ng-template>
+<p>
+(Všechny změny v dekretech se ukládají automaticky. Kliknutím na "Odeslat návrh dekretů" dáváte najevo, že návrh dekretů je
```

Comment:
This text is too long and similar as on the top, just write "Změny se ukládají stejně jako v akcích"

Reply:
Fixed.